### DISCUSSION TITLE
Does anyone know What the Private key file is for Scratch addons?

### DISCUSSION THREAD

**Discussion by TheRushxx21** [ScratchAddons/ScratchAddons#5237] [Help]
Does anyone know What the Private key file is for Scratch addons? I am Trying to Add Scratch addons as an extension to Microsoft edge, but it is blocked on this computer, and I am trying to find a work around. I think I found one though, but it is asking me for the Private Key File. Any idea of what that is?

**Comment by WorldLanguages** [comment]
Looks like you clicked "pack extension".
Try clicking "load unpacked" instead, and select the folder that has `manifest.json` inside it. (just downlaod the zip from https://github.com/ScratchAddons/ScratchAddons/archive/refs/tags/v1.28.0.zip)
In case "load unpacked" is unavailable due to blocking, tell me

**Reply by TheRushxx21** [reply]
Thanks for the suggestion but Load Unpacked is Blocked...

**Reply by BroJac5246** [reply]
Do you know why it's blocked? Is your browser managed?

"Load unpacked" is the proper way to load an extension, so you'll need to figure out how to use that.

**Reply by TheRushxx21** [reply]
My Browser is Managed And i cannot Unblock it, however (idk if it changes anything), I have not tried chrome yet

**Reply by TheRushxx21** [reply]
*Update: I now Have tried google and nothing changed

**Reply by BroJac5246** [reply]
> My Browser is Managed

That's your problem, your organization doesn't want you to load browser extensions that could allow you to bypass their restrictions.

**Reply by WorldLanguages** [reply]
Unfortunately, if that's blocked, there's not much you can do. You can use editor addons on turbowarp.org.
Is every extension blocked in your system, or is there a list of allowed ones?

**Comment by TheRushxx21** [comment]
The whole extention
________________________________

**Comment by Endermaster3690** [comment]
why is this in translation

**Reply by mxmou** [reply]
I don't know why the OP chose Translation. I changed the category to Help.
